react/src/views/Pages/Upload.jsx

```diff
@@ -13,8 +62,50 @@ class UploadPage extends React.Component {
     super(props);
 
     this.state = {
-      uploadURL: ""
+      uploadURL: "",
+      checked: [24, 22],
+      selectedValue: null,
+      selectedEnabled: "b",
+      vrtitle:""
     };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleChangeEnabled = this.handleChangeEnabled.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ selectedValue: event.target.value });
+  }
+  handleChangeEnabled(event) {
+    this.setState({ selectedEnabled: event.target.value });
+  }
+  handleToggle(value) {
+    const { checked } = this.state;
+    const currentIndex = checked.indexOf(value);
+    const newChecked = [...checked];
+
+    if (currentIndex === -1) {
+      newChecked.push(value);
+    } else {
+      newChecked.splice(currentIndex, 1);
+    }
+
+    this.setState({
+      checked: newChecked
+    });
+  }
+
+  updateVRTitle = e => {
+    this.setState(
+      {
+        vrtitle: e.value
+      }
+    )
+  }
```

Comment:
This function does't set the `vrtitle` attribute correctly. I added a debug statement to test this function out:
![image](https://user-images.githubusercontent.com/36905285/69400511-c22c2980-0ca6-11ea-9074-48bea3b56b1d.png)

This is what I see on the console when typing into the text field:
![image](https://user-images.githubusercontent.com/36905285/69400519-d1ab7280-0ca6-11ea-9fdd-c13f4cda89ae.png)